components/script/dom/webidls/HTMLPictureElement.webidl

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+ 
+// https://html.spec.whatwg.org/multipage/#htmlimageelement
```

Comment:
Shouldn't this link to https://html.spec.whatwg.org/multipage/#htmlpictureelement?

Reply:
ah, yes this should be https://html.spec.whatwg.org/multipage/#htmlpictureelement. I will do the required changes and update. Thanks.